platform-hub-api/app/services/git_hub_api_service.rb

```diff
@@ -0,0 +1,19 @@
+class GitHubApiService
+
+  def initialize options={}
+    @client = Octokit::Client.new(options)
+  end
+
+  def access_token=(access_token)
+    @client.access_token = access_token
+  end
+
+  def authorize_url client_id:, scope:, redirect_uri:, state:
+    @client.authorize_url(client_id, scope: scope, redirect_uri: redirect_uri, state: state)
+  end
+
+  def user
+    @client.user
+  end
+
+end
```

Comment:
Fair point. Initially I was expecting this to have more "wrapped up" functionality, but it didn't in the end. In any case these wrappers are really useful for mocking out in tests for the consumers. Otherwise I'd have to either a) dependency inject the `Octokit` class, or b) mock out the global class itself in a test. Mind you, it's not a big difference either way!

Also, I notice now that I'm using `Octokit` class directly in the consumer anyway. We should make this consistent either way, so I'll go with your suggestion, until we have a need for a separate wrapper 👍